bluesky/examples.py

```diff
@@ -154,8 +153,14 @@ def __getstate__(self):
         return (self.name, self._fields, self.read_attrs, self.conf_attrs,
                 self._monitor_intervals)
 
-    def trigger(self):
-        "No-op: returns a status object immediately marked 'done'."
+    def trigger(self, *, delay_time=0):
```

Comment:
Bad choice of words. I agree that ophyd is the reference implementation. With the recent changes, `Reader` is a useful public class in its own right, not just a base class. Is there a good reason to have it present a non-standard API? This `delay_time` seems like a init-time decision ("how should this fake device behave on trigger") and not something that should be passed through a Msg into the RE into the device at trigger-time.